website/static/js/pages/render-nodes.js

```diff
@@ -1,9 +1,15 @@
 'use strict';
 
 var $osf = require('js/osfHelpers');
+var $ = require('jquery');
 
 // model for components, due to simplicity did not create a new file
 var ComponentControl = {};
 
 // binds to component scope in render_nodes.mako
-$osf.applyBindings(ComponentControl, '#componentScope');
+$(document).ready(function() {
```

Comment:
Why does this now require delaying until the document is loaded, but didn't prior to the change?